2021-01-18 15:51:58,378-E:\software\Python3.8.3\UiAuto\common\run_case.py-[line:75]-40-find_case-ERROR-message:文件类型是：备份,不是用例文件!
2021-01-18 15:53:29,896-E:\software\Python3.8.3\UiAuto\common\general.py-[line:153]-40-element-ERROR-message:Message: 

2021-01-18 15:56:12,154-E:\software\Python3.8.3\UiAuto\common\run_case.py-[line:75]-40-find_case-ERROR-message:文件类型是：备份,不是用例文件!
2021-01-18 15:57:41,804-E:\software\Python3.8.3\UiAuto\common\general.py-[line:153]-40-element-ERROR-message:Message: 

2021-01-18 16:00:35,243-E:\software\Python3.8.3\UiAuto\common\run_case.py-[line:75]-40-find_case-ERROR-message:文件类型是：备份,不是用例文件!
2021-01-18 16:01:32,383-E:\software\Python3.8.3\UiAuto\common\run_case.py-[line:75]-40-find_case-ERROR-message:文件类型是：备份,不是用例文件!
2021-01-18 16:10:23,774-E:\software\Python3.8.3\UiAuto\common\run_case.py-[line:75]-40-find_case-ERROR-message:文件类型是：备份,不是用例文件!
2021-01-18 16:11:45,158-E:\software\Python3.8.3\UiAuto\common\run_case.py-[line:75]-40-find_case-ERROR-message:文件类型是：备份,不是用例文件!
2021-01-18 16:12:54,498-E:\software\Python3.8.3\UiAuto\common\run_case.py-[line:75]-40-find_case-ERROR-message:文件类型是：备份,不是用例文件!
2021-01-18 16:14:23,907-E:\software\Python3.8.3\UiAuto\common\general.py-[line:153]-40-element-ERROR-message:Message: 

2021-01-18 16:20:11,394-E:\software\Python3.8.3\UiAuto\common\run_case.py-[line:75]-40-find_case-ERROR-message:文件类型是：备份,不是用例文件!
2021-01-18 16:21:02,388-E:\software\Python3.8.3\UiAuto\common\run_case.py-[line:75]-40-find_case-ERROR-message:文件类型是：备份,不是用例文件!
2021-01-18 16:22:32,790-E:\software\Python3.8.3\UiAuto\common\run_case.py-[line:75]-40-find_case-ERROR-message:文件类型是：备份,不是用例文件!
2021-01-18 16:23:52,887-E:\software\Python3.8.3\UiAuto\common\run_case.py-[line:75]-40-find_case-ERROR-message:文件类型是：备份,不是用例文件!
2021-01-18 16:28:01,826-E:\software\Python3.8.3\UiAuto\common\run_case.py-[line:75]-40-find_case-ERROR-message:文件类型是：备份,不是用例文件!
2021-01-18 16:50:07,588-E:\software\Python3.8.3\UiAuto\common\run_case.py-[line:75]-40-find_case-ERROR-message:文件类型是：备份,不是用例文件!
2021-01-18 16:50:36,506-E:\software\Python3.8.3\UiAuto\common\general.py-[line:153]-40-element-ERROR-message:Message: 

2021-01-18 16:53:07,450-E:\software\Python3.8.3\UiAuto\common\run_case.py-[line:75]-40-find_case-ERROR-message:文件类型是：备份,不是用例文件!
2021-01-18 16:56:53,495-E:\software\Python3.8.3\UiAuto\common\run_case.py-[line:75]-40-find_case-ERROR-message:文件类型是：备份,不是用例文件!
2021-01-18 17:00:30,335-E:\software\Python3.8.3\UiAuto\common\run_case.py-[line:75]-40-find_case-ERROR-message:文件类型是：备份,不是用例文件!
2021-01-18 17:02:50,502-E:\software\Python3.8.3\UiAuto\common\run_case.py-[line:75]-40-find_case-ERROR-message:文件类型是：备份,不是用例文件!
2021-01-18 17:05:53,988-E:\software\Python3.8.3\UiAuto\common\run_case.py-[line:75]-40-find_case-ERROR-message:文件类型是：备份,不是用例文件!
2021-01-18 17:07:34,874-E:\software\Python3.8.3\UiAuto\common\run_case.py-[line:75]-40-find_case-ERROR-message:文件类型是：备份,不是用例文件!
2021-01-18 17:08:54,714-E:\software\Python3.8.3\UiAuto\common\run_case.py-[line:75]-40-find_case-ERROR-message:文件类型是：备份,不是用例文件!
2021-01-18 17:34:48,547-E:\software\Python3.8.3\UiAuto\common\run_case.py-[line:75]-40-find_case-ERROR-message:文件类型是：备份,不是用例文件!
2021-01-18 17:36:36,894-E:\software\Python3.8.3\UiAuto\common\run_case.py-[line:75]-40-find_case-ERROR-message:文件类型是：备份,不是用例文件!
2021-01-18 17:40:28,387-E:\software\Python3.8.3\UiAuto\common\run_case.py-[line:75]-40-find_case-ERROR-message:文件类型是：备份,不是用例文件!
2021-01-18 17:41:26,011-E:\software\Python3.8.3\UiAuto\common\run_case.py-[line:75]-40-find_case-ERROR-message:文件类型是：备份,不是用例文件!
2021-01-18 17:42:56,447-E:\software\Python3.8.3\UiAuto\common\run_case.py-[line:75]-40-find_case-ERROR-message:文件类型是：备份,不是用例文件!
